### DISCUSSION TITLE
Continued support for Adobe XD or Figma

### DISCUSSION THREAD

**Discussion by BenjaBobs** [adobe/react-spectrum#4716] [General]
Apologies if this is the wrong place to ask this question.

My team is currently strongly considering Spectrum because the foundation and design principles seem really well thought out, but tooling is quite important for our velocity.

From my understanding, Spectrum is used by Adobe internally and has first class support in Adobe XD through [Spectrum for Adobe XD plugin](https://spectrum.adobe.com/page/spectrum-xd-plugin/).
I am given to understand that Adobe XD is reaching [end of life](https://www.theverge.com/2023/6/22/23769586/adobe-xd-discontinued-shutting-down-figma-design-app).

I am wondering what this means for Spectrum, more specifically:
- Will Spectrum continue being Adobe's design language / component library
- What will happen to the Adobe XD plugin?
- Will there be an official Figma plugin?
  It seems to me that with the acquisition of Figma and the sunsetting of XD, Adobe will be pushing Figma as the new main design tool?
  Currently there is an [unofficial plugin](https://www.figma.com/community/file/1211274196563394418/Adobe-Spectrum-Design-System) but it doesn't have all the components, and when not officially supported by Adobe, updates are not guaranteed.
- If there won't be continued support for Adobe XD or Figma, what is the recommended way to use Spectrum collaboratively between both programmers and designers?

**Comment by devongovett** [comment]
Hi @BenjaBobs, sorry for the delay in responding here! React Spectrum will definitely continue to be maintained going forward. As the acquisition of Figma is ongoing at this time, I cannot speak too much about it. Hopefully the following helps to answer your questions:

**Will the Spectrum for XD Plugin have any updates and continue to be maintained?**
The Spectrum for XD Plugin is still available for download in the XD plugins marketplace. We don't have any updates on when the next release will be but we'll share an update as soon as possible. Thanks for your patience.

**Will Adobe publish a Spectrum library in the Figma Community?**
Until the acquisition closes, Adobe and Figma will continue to operate as separate companies. After it closes, we look forward to engaging with the XD and Figma communities to understand the ways we can add value.
